helm/cluster-vsphere/values.yaml

```diff
@@ -51,6 +51,11 @@ osUsers:
 - name: "giantswarm"
   sudo: "ALL=(ALL) NOPASSWD:ALL"
 
+deploy:
+  cni: true
+  cpi: true
+  csi: true
```

Comment:
This comment https://github.com/giantswarm/cluster-vsphere/pull/26/files#r1128572002 applies here as well.